.github/workflows/ansible-lint.yml

```diff
@@ -0,0 +1,15 @@
+---
+name: Reusable workflow for ansible-lint
+
+on:
+  workflow_call:
+
+jobs:
+  ansible-lint:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+  
+      - name: Run ansible-lint
+        uses: ansible/ansible-lint@main
```

Comment:
should we reference `main` here? I'd prefer to use a fixed version and let renovate do updates when they are needed.

Reply:
good idea, done.